xml/troubleshooting.xml

```diff
@@ -1320,7 +1320,7 @@
       </para>
       <variablelist>
        <varlistentry>
-        <term><filename>/etc/resolv.conf</filename>
```

Comment:
Stick to /etc/resolv.conf when troubleshooting. This allows to check the presence of the symlink. Maybe we should mention it should be a link and where it points too.